examples/snowflake_agent/snowflake_agent/snowflake_agent_example_usage.py

```diff
@@ -4,87 +4,122 @@
 #
 # This example shows how to use the `SnowflakeTask` to execute a query in Snowflake.
 #
-# To begin, import the required libraries.
 # %%
-from flytekit import kwtypes, workflow
+import pandas as pd
+from flytekit import ImageSpec, Secret, StructuredDataset, kwtypes, task, workflow
 from flytekitplugins.snowflake import SnowflakeConfig, SnowflakeTask
 
-# %% [markdown]
-# Instantiate a {py:class}`~flytekitplugins.snowflake.SnowflakeTask` to execute a query.
-# Incorporate {py:class}`~flytekitplugins.snowflake.SnowflakeConfig` within the task to define the appropriate configuration.
-# %%
-snowflake_task_no_io = SnowflakeTask(
-    name="sql.snowflake.no_io",
-    inputs={},
-    query_template="SELECT 1",
-    output_schema_type=None,
+image = ImageSpec(
+    packages=[
+        "flytekitplugins-snowflake",
+        "pandas",
```

Comment:
need pyarrow as well

Reply:
Let me mention this issue in `linear`, for every doc using `strucutred dataset` , the imagespec will fail,
we should be careful when doing dependency cleanup.